src/Form/ConfigureProfileForm.php

```diff
@@ -394,9 +394,7 @@ public static function getDependencies(array $module_list, $only_dependencies =
       return TRUE;
     }
 
-    // Add dependencies to the list. The new modules will be processed as
-    // the while loop continues.
-    while (list($module) = each($module_list)) {
+    foreach (array_keys($module_list) as $module) {
```

Comment:
Are you sure that is the same?

The `$module_list` is modified within the loop body.